pyproject.toml

```
@@ -1,3 +1,40 @@
 [build-system]
-requires = ['setuptools >= 40.8.0', 'wheel']
-build-backend = 'setuptools.build_meta'
+requires = ["flit_core >=3.2"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "packaging"
+description = "Core utilities for Python packages"
+version = "21.4.dev0"
+readme = "README.rst"
+requires-python = ">=3.7"
+authors = [{name = "Donald Stufft", email = "donald@stufft.io"}]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Apache Software License",
+  "License :: OSI Approved :: BSD License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+urls.Source = "https://github.com/pypa/packaging"
+dependencies = [
+  "pyparsing>=2.0.2,!=3.0.5",
+]
+
+
+[tool.flit.sdist]
+include = ["LICENSE*", "tests/", "docs/", ".coveragerc", ".flake8", "mypy.ini", ".pre-commit-config.yaml"]
```

Let’s exclude everything other than the licenses, and stick to flit’s defaults?

---

We do need at least tests. Tests are very important to include in the SDist, distribution packages and conda-forge need the tests folder. But all the other stuff is not needed as far as I know.

```suggestion
include = ["LICENSE*", "tests/"]
```

---

As I understand it, the license and readme files are automatically included when references as files in the metadata.  (worth double-checking though)

---

The license is not. See the discussion in the corresponding issue and https://github.com/pypa/flit/issues/549